Fix RFC3164 (legacy-syslog) parser expecting spaces after `<PRI>` (#3718)
tenzir/tenzir: changelog/next/bug-fixes/3718--legacy-syslog-pri-spaces.md

> RFC 3164 syslog parser no longer requires the `PRI`-field
> (part in <angle brackets> in the beginning of a message)
> to be followed by whitespace.

Reviewer: ```suggestion
The RFC 3164 syslog parser no longer requires a whitespace after the `PRI`-field
(part in <angle brackets> in the beginning of a message).
```
